lib/pka.c

```diff
@@ -1694,6 +1748,166 @@ int pka_modular_inverse(pka_handle_t   handle,
     return pka_submit_cmd(handle, user_data, CC_MODULAR_INVERT, &operands);
 }
 
+int pka_is_mont_ecdh_canonical(ecc_mont_curve_t* curve,

```

Comment:
I thought this function would be implemented by the test framework; if the curve is canonical then the test framework or the upper layer application calls the API with the relevant flag, e.g. is_canonical=true|false

Reply:
Okay I see. If this is an internal call (static) then it is fine with me :)